src/editors/radio.js

```diff
@@ -55,6 +55,7 @@ JSONEditor.defaults.editors.radio = JSONEditor.defaults.editors.select.extend({
 
     var radioContainerWrapper = this.theme.getContainer();
     radioContainerWrapper.appendChild(this.radioContainer);
+    radioContainerWrapper.dataset.containerFor = 'radio';
```

Comment:
You don't need the dataset part anymore?

Reply:
Theoretically, I don't 😄 I'm trying to refactor radio, let's discuss in #205

Reply:
Moved dataset part to afterInputReady()